src/optimizer/query_planner.c

```diff
@@ -9853,77 +9853,80 @@ static double
 qo_all_some_in_selectivity (QO_ENV * env, PT_NODE * pt_expr)
 {
   PRED_CLASS pc_lhs, pc_rhs;
-  double list_card = 0, icard;
+  double list_card = 0.0, icard = 0.0;
+  double equal_selectivity = 1.0;
   PT_NODE *lhs;
-  double equal_selectivity, in_selectivity, selectivity;
+  PT_NODE *arg1, *arg2;
+
+  /* To avoid repeated dereferencing */
+  arg1 = pt_expr->info.expr.arg1;
+  arg2 = pt_expr->info.expr.arg2;
 
   /* determine the class of each side of the range */
-  pc_lhs = qo_classify (pt_expr->info.expr.arg1);
-  pc_rhs = qo_classify (pt_expr->info.expr.arg2);
+  pc_lhs = qo_classify (arg1);
+  pc_rhs = qo_classify (arg2);
 
   /* The only interesting cases are: attr IN set or (attr,attr) IN set or attr IN subquery */
   if ((pc_lhs == PC_MULTI_ATTR || pc_lhs == PC_ATTR) && (pc_rhs == PC_SET || pc_rhs == PC_SUBQUERY))
     {
       if (pc_lhs == PC_MULTI_ATTR)
 	{
-	  lhs = pt_expr->info.expr.arg1->info.function.arg_list;
-	  equal_selectivity = 1;
-	  for ( /* none */ ; lhs; lhs = lhs->next)
+	  for (lhs = arg1->info.function.arg_list; lhs; lhs = lhs->next)
 	    {
 	      /* get index cardinality */
 	      icard = qo_index_cardinality (env, lhs);
-	      if (icard != 0)
+	      if (icard > 0.0)
 		{
-		  selectivity = (1.0 / icard);
+		  equal_selectivity /= icard;
```

Comment:
선택도 공식이 1/NDV입니다. 이전 작성이 가독성이 더 좋아 보이네요. 아래에 하나 더 있습니다.
```suggestion
		  equal_selectivity *= (1.0 / icard);
```

Reply:
맞습니다. 이전처럼 작성하는 것이 가독성이 좋아보이네요. 수정하겠습니다.